test/scene-border-distance.test.ts

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
This is the only new test. Changes to all other `.test.ts` files are refactoring only.